cantabular-import/scs.sh

```diff
@@ -39,32 +46,55 @@ DP_FRONTEND_DATASET_CONTROLLER_DIR="$DIR/dp-frontend-dataset-controller"
 ZEBEDEE_DIR="$DIR/zebedee"
 ZEBEDEE_GENERATED_CONTENT_DIR=${zebedee_root}
 
-ACTION=$1
+EXPECTED_RUNNING_SERVICES=35
+NUMBER_OF_RETRIES=3
 
 ##################### FUNCTIONS ##########################
 logSuccess() {
-    echo -e "$GREEN ${1} $RESET"
+    echo -e "$GREEN${1}$RESET"
+}
+
+logWarning() {
+    echo -e "$YELLOW${1}$RESET"
 }
 
 logError() {
-    echo -e "$RED ${1} $RESET"
+    echo -e "$RED${1}$RESET"
+}
+
+isGumInstalled() {
+   command -v gum &> /dev/null
 }
 
 splash() {
-    echo "Start Cantabular Services (SCS)"
-    echo ""
-    echo "Simple script to run cantabular import service locally and all the dependencies"
-    echo ""
-    echo "List of commands: "
-    echo "   chown     - change the service '.go' folder permissions from root to the user and group."
-    echo "               Useful for linux users."
-    echo "   clone     - git clone all the required GitHub repos"
-    echo "   down      - stop running the containers via docker-compose"
-    echo "   init-db   - preparing db services. Run this once"
-    echo "   help      - splash screen with all these options"
-    echo "   pull      - git pull the latest from your remote repos"
-    echo "   setup     - preparing services. Run this once, before 'up'"
-    echo "   up        - run the containers via docker-compose"
+    if isGumInstalled
+    then
+        cd "$DP_CANTABULAR_IMPORT_DIR"
+        gum style \
+        	--foreground 212 --border-foreground 57 --border double \
+            --align center --width 50 --margin "1 2" --padding "2 4" \
+        	'Start Cantabular Services (SCS)' \
+        	'' \
+        	'Simple script to run cantabular import service locally and all the dependencies'
+
+        cat scs-helper.txt | gum choose --no-limit | gumOptions $(awk '{print $1}')
+
+    else
+        echo "Start Cantabular Services (SCS)"
+        echo ""
+        echo "Simple script to run cantabular import service locally and all the dependencies"
+        echo ""
+        echo "List of commands: "
+        echo "   chown           - change the service '.go' folder permissions from root to the user and group. Useful for linux users."
+        echo "   clone           - git clone all the required GitHub repos"
+        echo "   fe-assets       - generate Cantabular FE assets"
+        echo "   init-db         - preparing db services. Run this once"
+        echo "   pull [branch]   - by default it will pull the latest from the current branch. Optionally, provide a branch to pull from, e.g., 'scs pull develop'"
```

Comment:
This is not an issue, but will be tempted to add an option where I can pass a repo name/names and that repo will not be pulled for develop/main something like 'scs pull develop -i dp-cantabular-filter-flex-api' 
This will resolve the issue of keeping your local work branch local. Its making a humble script that started with helping bare minimum stuff full fledged application :smiley: